include/nbl/builtin/hlsl/bda/bda_accessor.hlsl

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2018-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_BUILTIN_HLSL_BDA_ACCESSOR_INCLUDED_
+#define _NBL_BUILTIN_HLSL_BDA_ACCESSOR_INCLUDED_
+
+#include "nbl/builtin/hlsl/glsl_compat/core.hlsl"
+#include "nbl/builtin/hlsl/bda/__ptr.hlsl"
+
+namespace nbl 
+{
+namespace hlsl
+{
+namespace bda
+{
+
+template<typename T>
+struct BdaAccessor
+{
+    static BdaAccessor<T> create(const uint64_t addr)
```

Comment:
I'd create from a `_bda::_ptr<T>` instead of `uint64_t`

Reply:
Fixed